### PR TITLE
Add SX Network Mainnet

### PR DESCRIPTION
**Pull request by dankostiuk**
Fixes # .

Changes proposed in this pull request:
- add SX Network mainnet

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -693,6 +693,23 @@
...
+    "multicall": "0x834a005DDCF990Ba1a79f259e840e58F2D14F49a",
+    "rpc": [
+      "https://rpc.sx.technology"
+    ],
+    "ws": [
+      "wss://rpc.sx.technology/ws"
+    ],
+    "explorer": "https://explorer.sx.technology",
+    "start": 2680605,
+    "logo": "ipfs://QmSXLXqyr2H6Ja5XrmznXbWTEvF2gFaL8RXNXgyLmDHjAF"
```
Comment (ChaituVR) [inline src/networks.json:711]: @dankostiuk Did you pin this image somewhere? not able to resolve this , i would suggest to use a service like pinnata or fleek to pin this image to IPFS

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json (hunk 1 above)
Comment (ChaituVR) [reply src/networks.json:711]: Everything else looks good

**Comment by mrwillis** [comment]
@ChaituVR are there any other changes you'd need to get this merged?

**Comment by ChaituVR** [comment]
Hey @dankostiuk looks good, can you update the branch plz

**Review by bonustrack — APPROVED** [review]

**Comment by ChaituVR** [comment]
Sorry for not noticing this @dankostiuk Can you plz update the branch again, or click on Allow update from maintainers so i can update when we merge this😅

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**